jujugui/static/gui/src/app/components/profile/invoice-list/test-invoice-list.js

```diff
@@ -0,0 +1,142 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+
+'use strict';
+const React = require('react');
+const shapeup = require('shapeup');
+
+const BasicTable = require('../../basic-table/basic-table');
+const ProfileInvoiceList = require('./invoice-list');
+
+const jsTestUtils = require('../../../utils/component-test-utils');
+
+describe('Invoice Bundle List', function() {
+  const rawInvoiceData = `[{
+  	"number": "100001",
+  	"status": "Paid",
+  	"chargedTo": "card ending 1234",
+  	"date": "12/05/2017"
+  }, {
+  	"number": "100002",
+  	"status": "Declined: will retry",
+  	"chargedTo": "card ending 1231",
+  	"date": "12/05/2017"
+  }]`;
+
+  const invoices = JSON.parse(rawInvoiceData);
```

Comment:
To avoid any accidental modification of this object, you'll want to do this in a beforeEach